daemon/config.kv.sh

```diff
@@ -9,6 +9,16 @@ function get_mon_config {
 
   CLUSTER_PATH=ceph-config/${CLUSTER}
 
+  # making sure the root dirs are present for the confd to work
+  kviator --kvstore=${KV_TYPE} --client=${KV_IP}:${KV_PORT} put ${CLUSTER_PATH}/auth/_ _ > /dev/null 2>&1
```

Comment:
PS: _ doesn't actually show as a key in the store when listing them. it does come up on a get call but does not show in the confd file

I tried this trick: https://github.com/kelseyhightower/confd/issues/273#issuecomment-149900330 but that didn't help: it still confused confd when path did not exist.